src/apps/monero/protocol/key_image_sync.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Author: Dusan Klinec, ph4r05, 2018
+
+from trezor import log
+
+from apps.monero.controller import wrapper as twrap
+
+
+class KeyImageSync(object):
```

Comment:
`class KeyImageSync:` would be best here.

Reply:
I was wondering, do you prefer Py3-only classes? Extending object is my habit from writing Py23 compatible code but I think in Py3 it is not needed, right?

Reply:
Right. Also micropython is Py3 only, so it makes sense.